tests/CoreFX/TopN.CoreFX.x64.Unix.issues.json

```diff
@@ -1,48 +1,1808 @@
-// The expected format of this file can be found at https://github.com/dotnet/coreclr/blob/master/Documentation/building/testing-with-corefx.md
 [
     {
-        "name": "System.Collections.Tests",
+        "name": "System.Net.HttpListener.Tests",
+        "exclusions": {
+            "namespaces": null,
+            "classes": null,
+            "methods": [
+                {
+                    "name": "System.Net.Tests.HttpRequestStreamTests.Read_NullBuffer_ThrowsArgumentNullException",
+                    "reason": "Xunit.Sdk.ThrowsException Assert.Throws() Failure\\nExpected: typeof(System.ArgumentNullException)\\nActual:   typeof(System.NullReferenceException): Object reference not set to an instance of an object."
+                },
+                {
+                    "name": "System.Net.Tests.HttpResponseStreamTests.Write_NullBuffer_ThrowsArgumentNullException",
+                    "reason": "Xunit.Sdk.ThrowsException Assert.Throws() Failure\\nExpected: typeof(System.ArgumentNullException)\\nActual:   typeof(System.NullReferenceException): Object reference not set to an instance of an object."
+                }
+            ]
+        },
+        "enabled": true
+    },
+    {
+        "name": "System.Reflection.Context.Tests",
+        "exclusions": {
+            "namespaces": null,
+            "classes": null,
+            "methods": [
+                {
+                    "name": "System.Reflection.Context.Tests.CustomPropertyInfoTests.GetCustomAttributesDataTest",
+                    "reason": "System.PlatformNotSupportedException System.PlatformNotSupportedException : Customized reflection contexts are only supported on .NET Framework."
```

Comment:
This exception means that we are using wrong corefx bits (.NET Framework bits instead of netcoreapp30 bits) when running the tests. It would be nice to fix the problem with the wrong corefx bits - there seems to be a lot of the test exclusion because of this underlying problem.

Reply:
This should fix the failures here - https://github.com/dotnet/coreclr/pull/18760 .